std/math/operations.d

```diff
@@ -1717,7 +1717,7 @@ if (isFloatingPoint!T)
     bool negative;
 }
 
-FloatingPointBitpattern!T extractBitpattern(T)(T val)
+FloatingPointBitpattern!T extractBitpattern(T)(T val) @trusted
```

Comment:
I still think that this should use `@trused` lambdas but as this is a commit that is from stable, I'll let this go.